py/examples/combobox.py

```diff
@@ -20,13 +20,13 @@ async def serve(q: Q):
         ]
     else:
         q.page['example'] = ui.form_card(box='1 1 4 10', items=[
-            ui.combobox(name='combobox', label='Enter or choose a color', placeholder='Color...', value='Blue',
+            ui.combobox(name='combobox', label='Enter or choose a color', placeholder='Color...', value='Black',
```

Comment:
Agree with everything you said (by free text I meant the user can type whatever and that whatever gets added to the options).

> What I would be concerned about is if people are already using Wave Combobox like that and introducing this new behavior would break old apps.

Exactly. That's why I am suggesting keeping the old behavior and adding a minor enhancement - if the value is not among the options, add the new option with the value. We do that already in `onChange` IIRC.

That way, we do not break apps and have the correct combobox behavior.

Reply:
Added minor feature and tests for it.